File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int p = 1, zero = 0;
        for (int i : nums) {
            if (i == 0) {
                zero++;
                continue;
            } else {
                p *= i;
            }
        }
        if (zero >= 2) {
            for (int i = 0; i < nums.length; i++)
                nums[i] = 0;
        } else if (zero == 0) {
            for (int i = 0; i < nums.length; i++)
                nums[i] = p / nums[i];
        } else {
            for (int i = 0; i < nums.length; i++) {
                if (nums[i] == 0)
                    nums[i] = p / (nums[i] + 1);
                else
                    nums[i] = 0;

            }
        }
        return nums;

    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        s=""
        for i in nums:
            s=s+str(i)
        a=s.split('0')
        maxa=0
        for i in a:
            if len(i)>maxa:
                maxa=len(i)
        return maxa
